Makefile

```diff
@@ -134,7 +134,7 @@ endif
 
 .PHONY: build
 build: generate fmt vet ## Build manager binary.
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build $(GOFLAGS) -mod=mod -o bin/manager main.go
```

Comment:
This change is unrelated and should go as a separate commit

Reply:
That's true but since all this work is driven by the s390x enablement, it is acceptable to have this commit as preparatory fix.